script/upload

```diff
@@ -46,10 +43,16 @@ def upload(target_arch):
   else:
     platform = PLATFORM_KEY
 
-  upload_files = glob.glob('libchromiumcontent*.tar.bz2') + glob.glob('native-mksnapshot.tar.bz2')
-  s3put(bucket, access_key, secret_key, SOURCE_ROOT,
-        'libchromiumcontent/{0}/{1}/{2}'.format(platform, target_arch, commit),
-        upload_files)
+  if args.upload_packaged_src:
+    upload_files = glob.glob('src.tar.bz2')
```

Comment:
What does `glob.glob` do here?

Reply:
Good point, its not really needed - I'll change it